specs/capella/fork.md

```diff
@@ -129,9 +129,8 @@ def upgrade_to_capella(pre: bellatrix.BeaconState) -> BeaconState:
         # Execution-layer
         latest_execution_payload_header=latest_execution_payload_header,
         # Withdrawals
-        withdrawal_queue=[],
         next_withdrawal_index=WithdrawalIndex(0),
-        next_partial_withdrawal_validator_index=ValidatorIndex(0),
+        latest_withdrawal_validator_index=ValidatorIndex(0),
```

Comment:
This will start sweeping from validator_index = 1. It's a bit ugly, but putting here `ValidatorIndex(len(state.validators) - 1)` is uglier. I was tempted to put `latest_withdrawal_validator_index = 7653`.

Reply:
That can't work without an ugly hack :). Leaving it as is. Going through your review now. Thanks!